Tests/ManifestSerializer/JSONSerializationTests.swift

```diff
@@ -0,0 +1,122 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright 2015 - 2016 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See http://swift.org/LICENSE.txt for license information
+ See http://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+
+@testable import ManifestSerializer
+import PackageDescription
+import XCTest
+
+class JSONSerializationTests: XCTestCase {
+    
+    func testSimple() {
+        let package = Package(name: "Simple")
+        let exp = NSMutableDictionary.withNew { (dict) in
+            dict["name"] = "Simple"
+            fillWithEmptyArrays(keyNames: ["dependencies", "testDependencies", "exclude", "package.targets"], dict: dict)
+        }
+        assertEqual(package: package, expected: exp)
+    }
+    
+    func testDependencies() {
+        let deps: [Package.Dependency] =
+            [
+                .Package(url: "https://github.com/apple/swift.git", majorVersion: 3),
+                .Package(url: "https://github.com/apple/llvm.git", majorVersion: 2)
+        ]
```

Comment:
Yeah, but Xcode keeps reindenting it in a way that stretches the line way too to the right. And I use automatic reindent all the time :(